src/pages/brand.tsx

```diff
@@ -0,0 +1,446 @@
+            style={{ background: "white" }}
+          />
+        </div>
+        <div className="dos-donts">
+          <ul className="do">
+            <li>
+              <em>Do</em> prefer Rhodamine on a white background, our default
+              color combination.
+            </li>
+            <li>
+              <em>Do</em> otherwise prefer use the white logo on most
+              backgrounds unless contrast requires use of black.
+            </li>
+          </ul>
+        </div>
+        <div className="color-combos">
+          <img
+            src="/img/brand/logos/logo-black.svg"
+            style={{ background: "var(--rhodamine)" }}
+          />
+          <img src="/img/logo.svg" style={{ background: "#00f0ff" }} />
+          <img
+            src="/img/logo.svg"
+            style={{
+              background:
+                "url(/img/brand/logo-dont/dont-complex-background.jpg)",
+              backgroundSize: "cover",
+            }}
+          />
+          <img
+            src="/img/brand/logo-dont/dont-color.svg"
+            style={{ background: "white" }}
+          />
+          <img
+            className="filter"
+            src="/img/brand/logo-dont/dont-gradient.svg"
+            style={{ background: "white" }}
+          />
+          <img
+            src="/img/brand/logo-dont/dont-color-wordmark.svg"
+            style={{ background: "white" }}
+          />
+        </div>
+        <div className="dos-donts">
+          <ul className="dont">
+            <li>
+              <em>Don't</em> use black on a background of Rhodamine.
+            </li>
+            <li>
+              <em>Don't</em> use Rhodamine on a color or patterned background;
+              use white instead.
+            </li>
+            <li>
+              <em>Don't</em> use different colors or apply gradients to the
+              logo.
+            </li>
+            <li>
+              <em>Don't</em> use different colors for elements of the logo.
+              Always use the same color for the hexagraph and wordmark.
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section className="agree-actions-container bottom">
+        <AgreeActions />
+      </section>
+    </Layout>
+  )
+}
+
+function AgreeActions() {
+  const [agree, setAgree] = useState(false)
+  return (
+    <div className="agree-actions">
+      <label>
+        <input
+          type="checkbox"
+          checked={agree}
+          onChange={event => setAgree(event.target.checked)}
+        ></input>
+        I have read and accept the brand and trademark guidelines.
+      </label>
+      <a
+        {...(!agree
+          ? { className: "disabled" }
+          : { href: "/img/brand/graphql-brand-assets.zip" })}
+      >
+        Download Assets
+      </a>
+      <a
+        {...(!agree
+          ? { className: "disabled" }
+          : { href: "mailto:info@graphql.org?subject=Trademark+Request" })}
```

Comment:
Support for `+` in `mailto:` links is unreliable and will often carry the `+` through into the subject of the relevant mail client - this happens for the default iOS mail client for example. Use percent encoding instead:

```suggestion
          : { href: "mailto:info@graphql.org?subject=Trademark%20Request" })}
```

Reply:
TIL! This appears in a few spots so let me resolve them all